go/kbfs/libkbfs/journal_manager.go

```diff
@@ -383,7 +384,8 @@ func (j *JournalManager) makeFBOForJournal(
 		return nil
 	}
 
-	_, _, err = j.config.KBFSOps().GetRootNode(ctx, handle, data.MasterBranch)
+	j.log.CDebugf(ctx, "Making conflict FBO")
+	_, _, err = j.config.KBFSOps().GetRootNode(ctx, handle, data.MasterBranch /*branch*/)
```

Comment:
did you mean to use `branch` here instead of `data.MasterBranch`?

Reply:
Ooooops.  Fixed, thanks.  Also removed a couple debugging lines.